lumen/ai/agents.py

```diff
@@ -41,6 +41,8 @@
 
 FUZZY_TABLE_LENGTH = 10
 
+pn.chat.ChatStep.min_width = 350
```

Comment:
Totally on board with this but I'm not convinced this is the best place to configure this. That said I'm not sure where would be better.

Reply:
Would a config file be better?